### DISCUSSION TITLE
Date and Time not parsing while using custom NER Model

### DISCUSSION THREAD

**Discussion by sushamasivananthan** [explosion/spaCy#8002] [Help: Coding & Implementations] [answered]
Hi Team,
I'm trying to create a custom NER model to parse informations like Invoice number ,date, time and amount from invoices.

I have used around 50K+ records to train the model, but after training the model can detect the invoice number and tax id etc. but it is unable to detect date and time if there is even a slight change in date format(for example dd/mm/yy instead of dd/mm/yyyy).
Is there any option to combine pre-trained spacy model with my custom NER model. If so what code change should I do?

```
Environment:

spacy  : 3.0.6
OS: linux 
Python: 3.8.5
```

**### I have given my sample data set below.**

`NEW_TRAIN=[('UID:1234 17:28:35 29.2.20 ', {'entities': [(0, 8, 'UID'), (9, 17, 'time'), (18, 25, 'date')]})]`

**### My code looks like this:**

```
import pandas as pd
from tqdm import tqdm
import spacy
from spacy.tokens import DocBin

nlp = spacy.load("en_core_web_sm") # load a new spacy model
db = DocBin() # create a DocBin object

for text, annot in tqdm(NEW_TRAIN):
    #print(text)
    #print(annot)
#for text, annot in tqdm(TRAIN_DATA): # data in previous format
    doc = nlp.make_doc(text) # create doc object from text
    ents = []
    for start, end, label in annot["entities"]: # add character indexes
        #print(start)
        #print(end)
        #print(label)
        span = doc.char_span(start, end, label=label, alignment_mode="contract")
        #print(span)
        if span is None:
            
            print("Skipping entity")
        else:
            ents.append(span)
    doc.ents = ents # label the text with the ents
    db.add(doc)

db.to_disk("./train.spacy")
```

Also, I have created the config file from this link [https://spacy.io/usage/training] where I have mentioned language as **german** and component as **ner**

**### Mode training code looks like this**

`python3 -m spacy train config.cfg --output ./output --paths.train ./train.spacy --paths.dev ./train.spacy`

please help.

**Comment by svlandeg** [comment]
Hi! As a maintenance note: this is a good question for the discussions forum, so I'll move it there. The original issue will be closed and will link through to the open discussion thread.

**Comment by polm** [comment, marked as answer]
I'm going to lock this since it's an exact duplicate of #8001.
